### PR TITLE
NOISSUE - Add certs service and deployment charts

### PR DESCRIPTION
**Pull request by darkodraskovic**

### CODE REVIEW

File: charts/mainflux/values.yaml [hunk 1]
```diff
@@ -17,20 +17,20 @@ ingress:
   annotations: {}
   labels: {}
   # Uncomment this block for TLS support in public ingress
-  # hostname: ""
-  # tls:
-  #   hostname: ""
-  #   secret: "mainflux-server"
+  hostname: "ashvin01.mainflux.io"
```
Comment (mteodor) [inline charts/mainflux/values.yaml:20]: remove this addres

File: charts/mainflux/values.yaml [hunk 2]
```diff
@@ -67,7 +67,7 @@ mqtt:
   proxy:
     image:
       repository: "mainflux/mqtt"
-      tag: "0.11.0"
+      tag: "latest"
```
Comment (mteodor) [inline charts/mainflux/values.yaml:70]: latest or 0.11.0?

**Review by blokovi — CHANGES_REQUESTED** [review]
Add `postgresql-certs` in dependencies in Chart.yaml with `condition: certs.enabled`

File: charts/mainflux/values.yaml [hunk 3]
```diff
@@ -113,6 +113,26 @@ bootstrap:
   httpPort: 8182
   redisESPort: 6379
 
+certs:
+  enabled: true
```
Comment (nmarcetic) [inline charts/mainflux/values.yaml]: Disabled by default.

**Review by nmarcetic — CHANGES_REQUESTED** [review]

**Review by nmarcetic — APPROVED** [review]

**Review by blokovi — APPROVED** [review]

**Pull request merged**